CMakeLists.txt

```diff
@@ -18,6 +18,10 @@ else()
     add_compile_definitions(PATH_SEP='/')
 endif()
 
+if(DEFINED ENV{ZITI_DEBUG})
```

Comment:
testing a cmake cache variable instead of the environment would make more sense to me for an option that changes the build.